db/models/index.js

```diff
@@ -6,7 +6,21 @@
 
 const User = require('./user')
 const OAuth = require('./oauth')
+const Product = require('./product')
+const Record = require('./record')
+const Order = require('./order')
+const Review = require('./review')
 
 OAuth.belongsTo(User)
 User.hasOne(OAuth)
+
+Product.belongsTo(Record)
+Record.hasMany(Product)
+
+//Orders must belong to a user OR guest session (authenticated vs unauthenticated)
```

Comment:
Just associate it to the user and we can use a "null" user id for unauthenticated